pkg/config/setup/config.go

```diff
@@ -2284,12 +2284,12 @@ func setTracemallocEnabled(config pkgconfigmodel.Config) bool {
 	traceMallocEnabledWithPy2 := false
 	if pyVersion == "2" && wTracemalloc {
 		log.Warnf("Tracemalloc was enabled but unavailable with python version %q, disabling.", pyVersion)
-		wTracemalloc = false
 		traceMallocEnabledWithPy2 = true
+
+		// update config with the actual effective tracemalloc
+		config.Set("tracemalloc_debug", false, pkgconfigmodel.SourceAgentRuntime)
```

Comment:
In the previous code, the source was `pkgconfigmodel.SourceAgentRuntime`. Could it be an issue?

Reply:
I'm pretty sure it is not an issue, the only expected impact is that this value which was in the runtime layer now will stay in the file/env/default layer